strings/ai.txt

```diff
@@ -104,7 +104,6 @@ SEN 5241
 SHODAN
 SID 6.7
 Setaur
-Shrike
```

Comment:
This is kinda entirely unrelated? 
I am in awe at how this got into this PR, and it shows a whole lack of care to this whole PR.